thirdparty/hnswlib/hnswlib/neighbor_double_pop_list.h

```diff
@@ -0,0 +1,204 @@
+
+
```

Comment:
Do we still need `NeighborSet` in `neighbor.h`? Is it used anywhere else? If not, maybe clean it up?

Reply:
removed `NeighborSet`.
moved `NeighborSetDoublePopList` to `neighbor.h`.